pages/index.html

```diff
@@ -4,9 +4,57 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="stylesheets/style.css" rel="stylesheet"/>
+  <title>Joanna Paris Gems</title>
 </head>
 <body>
-  
+  <header>
+    <h1>
+    Joanna Paris Gems
+    </h1>
+  <h3>
```

Comment:
In semantic HTML, header tags should be used to indicate different sections of content, not as stylistic elements.  [Link](https://bighack.org/how-to-heading-structure-accessibility/)